Description: Change missing config instructions.
 If the server_name is not correctly set in the config, synapse will refuse
 to start and quit with an error message. This patch adds instructions on how
 to use dpkg-reconfigure to set them.
 .
 matrix-synapse (0.11.0-r2-1~1.gbpda43c7) UNRELEASED; urgency=medium
 .
   ** SNAPSHOT build @da43c7fd36272b89c27b5ac04aa7db111e305b97 **
 .
   * Imported Upstream version 0.11.0-r2
   * Add gbp.conf
Author: Erik Johnston <erikj@matrix.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- matrix-synapse-0.11.0-r2.orig/synapse/config/_base.py
+++ matrix-synapse-0.11.0-r2/synapse/config/_base.py
@@ -30,6 +30,11 @@ class ConfigError(Exception):
 MISSING_REPORT_STATS_CONFIG_INSTRUCTIONS = """\
 Please opt in or out of reporting anonymized homeserver usage statistics, by
 setting the `report_stats` key in your config file to either True or False.
+
+To set it run:
+
+    dpkg-reconfigure matrix-synapse
+
 """
 
 MISSING_REPORT_STATS_SPIEL = """\
@@ -44,6 +49,11 @@ Thank you.
 
 MISSING_SERVER_NAME = """\
 Missing mandatory `server_name` config option.
+
+To set it run:
+
+    dpkg-reconfigure matrix-synapse
+
 """
 
 
